client/src/lib/validateFunctions.js

```diff
@@ -0,0 +1,15 @@
+const isValidName = (name) => name.length >= 2;
```

Comment:
Kudos! Really nice that you outsourced this and wrote very neat little small functions 👍